lib/internal/Magento/Framework/Data/Collection.php

```diff
@@ -521,8 +521,8 @@ public function walk($callback, array $args = [])
      */
     public function each($objMethod, $args = [])
     {
-        foreach ($args->_items as $k => $item) {
-            $args->_items[$k] = call_user_func($objMethod, $item);
+        foreach ($this->getItems() as $item) {
+            call_user_func_array([$item, $objMethod], $args);
```

Comment:
`call_user_func_array` is quite slow compared to argument unpacking[1].

This should probably be changed to:

```php
$item->$objMethod(...$args);
```